agent/php_nrini.c

```diff
@@ -537,6 +537,28 @@ static PHP_INI_MH(nr_preload_framework_library_detection_mh) {
   return SUCCESS;
 }
 
+static PHP_INI_MH(nr_composer_per_process_detection_mh) {
+  int val;
+
+  (void)entry;
+  (void)NEW_VALUE_LEN;
+  (void)mh_arg1;
+  (void)mh_arg2;
+  (void)mh_arg3;
+  (void)stage;
+  NR_UNUSED_TSRMLS;
+
+  val = nr_bool_from_str(NEW_VALUE);
+
+  if (-1 == val) {
+    return FAILURE;
```

Comment:
I think it would be useful for the user to log when an invalid string was given for an option like this.
Something like
`nrl_warning("The value %s is not valid for <insert INI value name>, using default value");`

Reply:
I think it would be better to print out the `NEW_VALUE` so they can see the invalid string they used in the INI file.  The current change would just print out that "-1" is an invalid value?

Reply:
c43e3c01aee8b369d914a1f2ccf25f38d4742175